### PR TITLE
Add comments and docstring for the import image function

### PR DESCRIPTION
**Pull request by naved001**

### CODE REVIEW

File: ims/einstein/operations.py [hunk 1]
```diff
@@ -430,20 +430,42 @@ def list_all_images(self):
 
     @log
     def import_ceph_image(self, img):
+        """ Import an image in ceph to be used by BMI
```
Comment (chemistry-sourabh) [inline ims/einstein/operations.py]: New line here

Comment (chemistry-sourabh) [inline ims/einstein/operations.py]: There is a second import in same file so add some more specificity to the docstring and also probably document the second one also in this PR.

File: ims/einstein/operations.py [hunk 2]
```diff
@@ -430,20 +430,42 @@ def list_all_images(self):
...
+            # this is needed because, in ceph, you can only create clones from
+            # snapshots.
             self.fs.snap_image(ceph_img_name, constants.DEFAULT_SNAPSHOT_NAME)
             self.fs.snap_protect(ceph_img_name,
                                  constants.DEFAULT_SNAPSHOT_NAME)
+
+            # insert golden image name into bmi db
             self.db.image.insert(ceph_img_name, self.pid)
+
+            # get a name for our copy of the golden image
```
Comment (chemistry-sourabh) [inline ims/einstein/operations.py]: This line is a bit vague. Perhaps add an example like golden_image -> 1img1

**Comment by naved001** [comment]
@chemistry-sourabh Done!

File: ims/einstein/operations.py [hunk 3]
```diff
@@ -430,20 +430,47 @@ def list_all_images(self):
...
             self.fs.snap_image(ceph_img_name, constants.DEFAULT_SNAPSHOT_NAME)
             self.fs.snap_protect(ceph_img_name,
                                  constants.DEFAULT_SNAPSHOT_NAME)
+
+            # insert golden image name into bmi db
             self.db.image.insert(ceph_img_name, self.pid)
+
+            # get a name for our copy of the golden image. For instance an
+            # image in ceph called centos6.7, after cloning, will be a given
+            # a name like 4img1 based on the UUID
```
Comment (chemistry-sourabh) [inline ims/einstein/operations.py]: based on UID in config and image id in db.

**Comment by chemistry-sourabh** [comment]
Related to #45

**Comment by naved001** [comment]
@djfinn14 and @pgrosu, can either of you review this? it's a small PR and I am only adding comments for the import functions.

**Comment by naved001** [comment]
@pgrosu wants to review this over the weekend, hence the "do not merge" label.

**Comment by naved001** [comment]
@pgrosu, ping. I don't want to wait too long for a small PR like this which just has comments/doc-string.

**Comment by pgrosu** [comment]
Yes, this evening will do.

**Comment by naved001** [comment]
@pgrosu ping

**Pull request merged**